src/app/multi-editor/multi-editor.component.ts

```diff
@@ -264,6 +265,17 @@ export class MultiEditorComponent implements OnInit {
     this.changeDetectorRef.markForCheck();
   }
 
+  checkExistenceOfAction(userActions: UserActions): boolean {
+    let existingAction = false;
+    userActions.actions.forEach(action => {
+      if (action.mainKey) {
+        existingAction = true;
+      } else {
+        this.toastr.error('Please fill in the field for every action');
```

Comment:
this will show toast for every empty action, and showing toastr also done after calling this based on the result.

Maybe a leftover? if it's necessary should be called after this function is called based the result.